src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java

```diff
@@ -378,4 +379,36 @@ void testGetEventTypesAffectedByEndpoint() {
         assertEquals(ev1.getId(), eventTypes[0].getId());
     }
 
+    @Test
+    void testGetApplicationFacets() {
+        String tenant = "test";
+        String userName = "user";
+        String localIdentityHeaderValue = TestHelpers.encodeIdentityInfo(tenant, userName);
+        Header localIdentityHeader = TestHelpers.createIdentityHeader(localIdentityHeaderValue);
+        mockServerConfig.addMockRbacAccess(localIdentityHeaderValue, RbacAccess.READ_ACCESS);
+        given()
+                .header(localIdentityHeader)
+                .when()
+                .contentType(ContentType.JSON)
+                .get("/notifications/facets/applications?bundleName=insights")
+                .then()
+                .statusCode(200);
```

Comment:
Does the response contain some facets that could be checked here or is the response empty?

Reply:
It should have "policies" at least.